Simplenote/src/main/java/com/automattic/simplenote/utils/SafeThemeListPreference.kt

```diff
@@ -0,0 +1,42 @@
+package com.automattic.simplenote.utils
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.preference.ListPreference
+
+/**
+ * Custom ListPreference that handles invalid index values to prevent ArrayIndexOutOfBoundsException.
+ * This preference safely handles cases where the stored value doesn't match any of the available entries.
+ */
+class SafeThemeListPreference @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = androidx.preference.R.attr.preferenceStyle,
+    defStyleRes: Int = 0
+) : ListPreference(context, attrs, defStyleAttr, defStyleRes) {
+
+    override fun getSummary(): CharSequence? {
+        val value = getValue()
```

Comment:
@mzorz - is there a change that getValueI() can return null?  I'm can't tell if findIndexOfValue(value) handles that case.

Reply:
Great observation! exactly that case is handled in findIndexOfValue, so no worries 👍